packages/blade/src/components/Amount/Amount.tsx

```diff
@@ -274,24 +313,40 @@ const _Amount = ({
         alignItems="baseline"
         flexDirection="row"
       >
-        <BaseText
-          marginRight="spacing.1"
-          fontWeight={weight}
-          fontSize={currencyFontSize}
-          color={amountValueColor}
-          as={isReactNative ? undefined : 'span'}
-        >
-          {currencyPrefix}
-        </BaseText>
+        {currencyPosition === 'left' && (
+          <BaseText
+            marginRight="spacing.1"
+            fontWeight={weight}
+            fontSize={currencyFontSize}
+            color={amountValueColor}
+            as={isReactNative ? undefined : 'span'}
+            opacity={isAffixSubtle ? 0.5 : 1}
```

Comment:
We're using opacity instead of a color? Is that the correct usage? What if the amount component is placed on a non-white background. It will change actual color depending on the background then. Can you check with design once?